relay/sources/relayd/tests/files/config/main.conf

```diff
@@ -4,6 +4,10 @@ nodes_certs_file = "tests/keys/nodescerts.pem"
 node_id = "root"
 listen = "127.0.0.1:3030"
 
+# By default, the number of CPUs
+#core_threads = "4"
```

Comment:
This does not seem to match the None above

Reply:
It gives `None` when the key is not is the config. Once you use the configuration line, you have to give a precise number of thread.
We could add another option like a boolean "autoscale_core_threads" (or a magic value like 0, but I don't like it)